static/js/apps/nl_interface/app.tsx

```diff
@@ -49,7 +49,7 @@ export function App(): JSX.Element {
   function fetchData(paramsStr: string): void {
     setLoading(true);
     setParamsStr(paramsStr);
-    axios.get(`/nl/data?${paramsStr}`).then((resp) => {
+    axios.get(`/nl/data?q=${paramsStr}`).then((resp) => {
```

Comment:
remove change in 52?  Should add a comment that paramStr is a full url param string with all the args

Reply:
yes, it apparently had no effect either way? removed